docs/connectors/taps/google_analytics.rst

```diff
@@ -0,0 +1,79 @@
+
+.. _tap-google-analytics:
+
+Tap Google Analytic
+-----------
+
+
+Configuring what to replicate
+'''''''''''''''''''''''''''''
+
+PipelineWise configures every tap with a common structured YAML file format.
+A sample YAML for Google Analytic replication can be generated into a project directory by
+following the steps in the :ref:`generating_pipelines` section.
+
+Example YAML for ``tap-google-analytics``:
+
+.. code-block:: bash
+
+    ---
+
+    # ------------------------------------------------------------------------------
+    # General Properties
+    # ------------------------------------------------------------------------------
+    id: "google_analytics_sample"          # Unique identifier of the tap
+    name: "Google Analytics"               # Name of the tap
+    type: "tap-google-analytics"           # !! THIS SHOULD NOT CHANGE !!
+    owner: "somebody@foo.com"              # Data owner to contact
+
+
+    # ------------------------------------------------------------------------------
+    # Source (Tap) - Google Analytics connection details
+    # ------------------------------------------------------------------------------
+    db_conn:
+      # Method: OAuth authentication
+      view_id: "<view-id>"
+      start_date: "2010-01-01"  # specifies the date at which the tap will begin pulling data
+
+      oauth_credentials:
+        client_id: "<client-id>"
+        client_secret: "<oauth-client-id>"               # Plain string or vault encrypted
+        access_token: "<access-token>"                   # Plain string or vault encrypted
+        refresh_token: "<refresh-token>"                 # Plain string or vault encrypted
+
```

Comment:
can you please add `# key_file_location: "full-path-to-client_secrets.json"` with some comments that connecting via service accounts is also supported